asmcomp/asmgen.ml

```diff
@@ -266,7 +268,7 @@ type middle_end =
 let asm_filename output_prefix =
     if !keep_asm_file || !Emitaux.binary_backend_available
     then output_prefix ^ ext_asm
-    else Filename.temp_file "camlasm" ext_asm
+    else Filename.temp_file "camlasm" ext_asm (* Note: creates temporary file *)
```

Comment:
The comment (on a call to `Filename.temp_file`) is a bit of a tautology :-)

Reply:
Indeed; I removed it.